### PR TITLE
remove usage of six.b("literal")

### PR DESCRIPTION
**Pull request by a-detiste**
as a side effect the program would maybe run faster

### CODE REVIEW

**Review by tomato42 — CHANGES_REQUESTED** [review]

File: src/ecdsa/__init__.py [hunk 1]
```diff
@@ -90,7 +90,7 @@
     SECP160r1,
     Ed25519,
     Ed448,
-    six.b(""),
```
[the remark this answers is not in the document]
Comment (a-detiste) [reply src/ecdsa/__init__.py]: ok!

**Comment by a-detiste** [comment]
I think `black` makes some code better, some code worse.

I don't like it personally.

BAD:

```
             y_qB=int(
-                "990C57520812BE512641E47034832106BC7D3E8DD0E4C7F1136D70065"
-                "47CEC6A",
+                "990C57520812BE512641E47034832106BC7D3E8DD0E4C7F1136D70065" "47CEC6A",
                 16,
             ),
             x_Z=int(
-                "89AFC39D41D3B327814B80940B042590F96556EC91E6AE7939BCE31F3A"
-                "18BF2B",
+                "89AFC39D41D3B327814B80940B042590F96556EC91E6AE7939BCE31F3A" "18BF2B",
                 16,
             ),
             y_Z=int(
-                "49C27868F4ECA2179BFD7D59B1E3BF34C1DBDE61AE12931648F43E5963"
-                "2504DE",
+                "49C27868F4ECA2179BFD7D59B1E3BF34C1DBDE61AE12931648F43E5963" "2504DE",
                 16,
             ),
```

GOOD:

```
-            x_Z=int(
-                "312DFD98783F9FB77B9704945A73BEB6DCCBE3B65D0F967DCAB574EB", 16
-            ),
-            y_Z=int(
-                "6F800811D64114B1C48C621AB3357CF93F496E4238696A2A012B3C98", 16
-            ),
+            dA=int("7C4B7A2C8A4BAD1FBB7D79CC0955DB7C6A4660CA64CC4778159B495E", 16),
+            x_qA=int("B104A67A6F6E85E14EC1825E1539E8ECDBBF584922367DD88C6BDCF2", 16),
+            y_qA=int("46D782E7FDB5F60CD8404301AC5949C58EDB26BC68BA07695B750A94", 16),
+            dB=int("63976D4AAE6CD0F6DD18DEFEF55D96569D0507C03E74D6486FFA28FB", 16),
         )
```

**Comment by a-detiste** [comment]
(I had also fixed 2 duplicates of `sys` that pyflakes complained about)

**Comment by tomato42** [comment]
change those lines so that they're not two separate strings?

change from `...F1136D70065" "47CEC6A",` to `...F1136D7006547CEC6A",`?

**Comment by tomato42** [comment]
use `tox -e codeformat` to reformat according to project's black configuration, we don't use default black config

**Comment by tomato42** [comment]
Please [rebase](https://help.github.com/articles/about-git-rebase/) your PR on top of current master branch, then [force push](https://stackoverflow.com/a/10510482/462370) to the same branch as this PR is using.

Do not merge the `master` branch to this branch to update it.

**Review by tomato42 — APPROVED** [review]

**Comment by a-detiste** [comment]
thank you

**Pull request merged**